core/src/data/value/binary_op/decimal.rs

```diff
@@ -381,6 +427,17 @@ impl TryBinaryOperator for Decimal {
                     }
                     .into())
                 }),
+            U8(rhs) => lhs
+                .checked_rem(Decimal::from(rhs))
+                .map(|x| Ok(Decimal(x)))
+                .unwrap_or_else(|| {
+                    Err(ValueError::BinaryOperationOverflow {
+                        lhs: Decimal(lhs),
+                        operator: NumericBinaryOperator::Modulo,
+                        rhs: U8(rhs),
+                    }
+                    .into())
+                }),
```

Comment:
<img width="588" alt="image" src="https://user-images.githubusercontent.com/27040963/189530362-188bedd4-b945-4ad8-bc26-f888ce4772f2.png">

How about writing the same code as above?